.github/workflows/load-tests-job-configs/beam_LoadTests_Go_GBK_Dataflow_Batch_100b.txt

```diff
@@ -1,4 +1,3 @@
-###############################################################################
```

Comment:
Since we're updating `.txt` files as well, can we name it as `go_GBK_Dataflow_Batch_*.txt`?